src/lib/openapi/spec/admin-features-query-schema.ts

```diff
@@ -4,18 +4,20 @@ export const adminFeaturesQuerySchema = {
     $id: '#/components/schemas/adminFeaturesQuerySchema',
     type: 'object',
     additionalProperties: true,
+    description: 'Used to filter feature toggles from the admin-api',
     properties: {
         tag: {
             type: 'array',
             items: {
-                type: 'array',
-                items: {
-                    type: 'string',
-                },
+                type: 'string',
             },
+            description:
+                'Used to filter by tags. For each entry, a TAGTYPE:TAGVALUE is expected',
         },
         namePrefix: {
             type: 'string',
+            description:
+                'A case-insensitive prefix filter for the names of feature toggles',
```

Comment:
Can we add an example here? Something like this?

```suggestion
            description:
                'A case-insensitive prefix filter for the names of feature toggles',
            example: 'top_secret_',
```